lib/src/handle_access_token_response.dart

```diff
@@ -17,6 +17,54 @@ import 'authorization_exception.dart';
 /// amount of time.
 const _expirationGrace = const Duration(seconds: 10);
 
+/// Returns a function that that parses parameters from a response with either a JSON or URL-encoded body.
+Function parseJsonOrUrlEncoded(void validate(bool condition, String message)) {
```

Comment:
I was definitely considering making this a typedef; just didn't want to add any extraneous changes without getting a green light from you guys first.

I can make that change if you need me to